pom.xml

```diff
@@ -189,6 +196,17 @@
           <excludeFilterFile>spotbugs-exclude.xml</excludeFilterFile>
         </configuration>
       </plugin>
+      <plugin>
```

Comment:
Is this really required? I thought this is only required when generating a test jar, not when using one.

Reply:
I changed peass-version for dependency-module to ${peass.version} and removed the unnecessary maven-jar-plugin.